test/e2e/specs/customer_admin.go

```diff
@@ -233,15 +236,26 @@ var _ = Describe("Openshift on Azure customer-admin e2e tests [CustomerAdmin][Fa
 	It("should sync AAD admin group", func() {
 		syncSecret, err := sanity.Checker.Client.Admin.CoreV1.Secrets("openshift-infra").Get("aad-group-sync-config", metav1.GetOptions{})
 		Expect(err).ToNot(HaveOccurred())
-		id := "44e69b4e-2e70-42df-bb97-3a890730d7b0"
-		testUser := "testuserdisabled"
-		gid := syncSecret.Data["customerAdminGroupId"]
-		if strings.EqualFold(string(gid), id) {
-			// test the users
-			oca, err := sanity.Checker.Client.Admin.UserV1.Groups().Get("osa-customer-admins", metav1.GetOptions{})
+
+		var provider api.AADIdentityProvider
+		err = yaml.Unmarshal(syncSecret.Data["aad-group-sync.yaml"], &provider)
+		Expect(err).ToNot(HaveOccurred())
+
+		if provider.CustomerAdminGroupID != nil &&
+			*provider.CustomerAdminGroupID == "44e69b4e-2e70-42df-bb97-3a890730d7b0" {
+			var g *userv1.Group
+			err = wait.PollImmediate(2*time.Second, 10*time.Minute, func() (bool, error) {
```

Comment:
It will always take 10 minutes for this test to fail, which is sort of unfortunate, but not blocking.

Reply:
you're right, fixing